src/tools/how_to_model_slice.ts

```diff
@@ -0,0 +1,466 @@
+ - Use "variants" to provide style options for buttons (e.g., Primary/Secondary).
+ - If "select" is "document", use "customtypes"/"tags" to scope allowed documents.
+ - Do not model multiple adjacent buttons as separate fields; use one Link field with repeat: true.
+ 
+ ### ContentRelationship
+ 
+ {
+   "type": "ContentRelationship",
+   "config": {
+     "label": "Relationship label",
+     "customtypes": [
+       {
+         "id": "page",
+         "fields": [
+           "title",
+           { "id": "author", "fields": ["name"] }
+         ]
+       },
+       {
+         "id": "article",
+         "fields": ["title", "category"]
+       }
+     ],
+     "tags": ["featured"]
+   }
+ }
+ 
+ Notes:
+ - Use when you need a typed relationship to other Prismic documents.
+ - The customtypes array supports per-type field whitelisting via a nested "fields" array (strings or grouped field objects).
+ 
+ ### LinkToMedia
+ 
+ {
+   "type": "LinkToMedia",
+   "config": {
+     "label": "Media link label"
+   }
+ }
+ 
+ ### Boolean
+ 
+ {
+   "type": "Boolean",
+   "config": {
+     "label": "Boolean label"
+   }
+ }
+ 
+ ### Number
+ 
+ {
+   "type": "Number",
+   "config": {
+     "label": "Number label",
+     "placeholder": "Optional placeholder",
+     "min": 0,
+     "max": 100,
+     "step": 1
+   }
+ }
+ 
+ Notes:
+ - \`min\`, \`max\`, and \`step\` are optional.
+ 
+ ### Select
+ 
+ {
+   "type": "Select",
+   "config": {
+     "label": "Select label",
+     "placeholder": "Optional placeholder",
+     "options": ["Option A", "Option B", "Option C"],
+     "default_value": "Option A"
+   }
+ }
+ 
+ ### Date
+ 
+ {
+   "type": "Date",
+   "config": {
+     "label": "Date label",
+     "placeholder": "Optional placeholder"
+   }
+ }
+ 
+ ### Timestamp
+ 
+ {
+   "type": "Timestamp",
+   "config": {
+     "label": "Timestamp label",
+     "placeholder": "Optional placeholder"
+   }
+ }
+ 
+ ### Color
+ 
+ {
+   "type": "Color",
+   "config": {
+     "label": "Color label"
+   }
+ }
+ 
+ ### Embed
+ 
+ {
+   "type": "Embed",
+   "config": {
+     "label": "Embed label",
+     "placeholder": "Paste URL or embed code"
+   }
+ }
+ 
+ ### GeoPoint
+ 
+ {
+   "type": "GeoPoint",
+   "config": {
+     "label": "Location label"
+   }
+ }
+ 
+ ### Table
+ 
+ {
+   "type": "Table",
+   "config": {
+     "label": "Table label"
+   }
+ }
+ 
+ ### Group (repeatable set of fields)
+ 
+ {
+   "type": "Group",
+   "config": {
+     "label": "Group label",
+     "fields": {
+       "field_key": { /* any valid field shape from this section */ }
+     }
+   }
+ }
+ 
+ Notes:
+ - Use Group when you need a set of multiple fields to repeat together.
+ - Do not use variation-level "items".
+ - Do not use Group to model lists of links/buttons—prefer a single repeatable Link field.
+ 
+ ## Implementation Steps [MANDATORY]
+ 
+ 1) Create the directory \`src/slices/${sliceName}/\` if it doesn't exist
```

Comment:
❓ So finally it's better to let AI handle the creation? As it's deterministic, I feel, could it be faster / safer?

Reply:
yeah, seems the way to go, it's simple enough and avoid us to do anything other than given advice here, with minimal "code" logic